app/views/hub.nunjucks

```diff
@@ -17,7 +17,7 @@
       {% if gp.contact.telephone %}Telephone: {{ gp.contact.telephone }}<br />{% endif %}
       {% if gp.contact.email %}Email: {{ gp.contact.email }}<br />{% endif %}
       {% if gp.contact.fax %}Fax: {{ gp.contact.fax }}<br />{% endif %}
-      <a href="https://www.google.com/maps?q={{ gp.location.latitude }},{{ gp.location.longitude }}">See this surgery on a map</a>
+      <a href="https://www.google.com/maps/dir/{{ gp.name | safe | replace(" ", "+")}},{{ gp.address.addressLines | safe | replace(" ", "+") }},{{ gp.address.postcode | safe | replace(" ", "+")}}/@{{ gp.location.latitude }},{{ gp.location.longitude }},17z">See this surgery on a map</a>
```

Comment:
Can this link be generated outside of the view and added to the view model so this line would become more like `<a href="{{ gp.address.mapLink }}">...`?
It will be clearer to read and more easily tested.

Should the link be taking the map into directions mode? If not I'd suggest something a bit simpler e.g. https://maps.google.com/maps?q=<name, address, postcode> I'd leave off the lat/lon and zoom level and let Google decide what is best.